src/upcoming-components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -52,7 +28,8 @@ export interface Props {
  * import {AccordionPanel} from "@chanzuckerberg/eds";
  * ```
  *
- * TODO: update this comment with a description of the component.
+ * Accordian panel component to toggle between accordian content and active styling for the selected accordian,
+ * with adddtional styling passing in.
```

Comment:
How about something like..
```suggestion
 * Panel inside Accordion component for one section of content that can be shown and hidden by the user.
```
?